spk/python310/Makefile

```
@@ -44,6 +44,14 @@ PYTHON_LIB_DIR = lib/python$(SPK_VERS_MAJOR_MINOR)
 
 POST_STRIP_TARGET = python310_extra_install
 
+include ../../mk/spksrc.archs.mk
+
+# Enfore 32-bit object-mode on 32-bit platforms
+# as otherwise it fails to build on github-action
+ifneq ($(findstring $(ARCH),$(32bit_ARCHS)),$(ARCH))
+ENV = OBJECT_MODE=32
```

It's better not to overwrite the ENV (at least with #4949)
```suggestion
ENV += OBJECT_MODE=32
```

---

good catch.  But in the end this is not necessary.  I was trying figure out where the issue came from and that option really didn't work out (and was a shot in the dark).  thnx.